python/tidy/servo_tidy/tidy.py

```diff
@@ -19,6 +19,53 @@
 import sys
 from licenseck import MPL, APACHE, COPYRIGHT, licenses_toml, licenses_dep_toml
 import colorama
+import toml
+
+# Default configs
+config = {
+    "skip-check-length": False,
+    "skip-check-licenses": False,
+    "ignore": {
+        # Ignore hidden files by default
+        "files": ["./."],
+        # Ignore hidden directories by default
+        "directories": ["./."],
+        "packages": [],
+    }
+}
+
+# Load configs from servo-tidy.toml
+if os.path.exists("./servo-tidy.toml"):
```

Comment:
Looks like we're using `"servo-tidy.toml" throughout the code - maybe put that as a constant at the top (say, `CONFIG_FILE_PATH`?)